.github/workflows/test.yml

```diff
@@ -30,6 +30,7 @@ jobs:
         with:
           otp-version: ${{ env.OTP_VERSION }}
           elixir-version: ${{ env.ELIXIR_VERSION }}
+          experimental-otp: true
```

Comment:
Adding this to fix a CI issue, similar to what Micky faced [here](https://github.com/hanam1ni/google-crawler/pull/23/files#r546490492)

Can read more about it [in this issue](https://github.com/actions/setup-elixir/issues/31)